metrics-common/metrics-common-0.9.2-SNAPSHOT.ivy

```diff
@@ -43,5 +43,6 @@ under the License.
     <dependency org="org.slf4j" name="slf4j-log4j12" rev="1.7.14" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)">
         <artifact name="slf4j-log4j12" ext="jar"/>
     </dependency>
+    <dependency org="io.dropwizard.metrics" name="metrics-core" rev="3.2.3" conf="compile->compile(*),master(*);runtime->runtime(*)/>
```

Comment:
I am not sure if this library has been ELR-ed. We may not need to include this because we don't need this available other than build time to create a JAR.

Reply:
It is to make sure jars built with ivy have the same dependencies with jars built with maven pom.